include/RAJA/ListSegment.hxx

```diff
@@ -219,7 +220,17 @@ ListSegment::ListSegment(const T& indx)
   m_indx(0), m_indx_own(Unowned), m_len( indx.size() )
 {
    if ( !indx.empty() ) {
+#if defined(RAJA_USE_CUDA)
+      if ( cudaMallocManaged((void **)&m_indx, m_len*sizeof(Index_type),
```

Comment:
David,

Yes, we are a bit inconsistent.  I’ll change that and others if I find any to use cudaErrck and update the pull request.  I assume you’ll be fine with that change.

—Rich

From: David Poliakoff <notifications@github.com<mailto:notifications@github.com>>
Reply-To: LLNL/RAJA <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, May 23, 2016 at 9:38 AM
To: LLNL/RAJA <RAJA@noreply.github.com<mailto:RAJA@noreply.github.com>>
Cc: Rich Hornung <hornung1@llnl.gov<mailto:hornung1@llnl.gov>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [LLNL/RAJA] Fixed bug in template constructor for ListSegment so it works with UM. (#26)


In include/RAJA/ListSegment.hxx<https://github.com/LLNL/RAJA/pull/26#discussion_r64248464>:

> @@ -219,7 +220,17 @@ ListSegment::ListSegment(const T& indx)
>    m_indx(0), m_indx_own(Unowned), m_len( indx.size() )
>  {
>     if ( !indx.empty() ) {
> +#if defined(RAJA_USE_CUDA)
> +      if ( cudaMallocManaged((void **)&m_indx, m_len*sizeof(Index_type),


Generally when we use this idiom, we're using a cudaErrchk type of call, why not here?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/LLNL/RAJA/pull/26/files/afe5d408c1c345c07489ba16c23e68b0eb091c30#r64248464>